www/js/components/point-card.js

```diff
@@ -18,19 +92,70 @@ export default class PointCard extends Component {
       height: '176px',
       background: 'url(' + this.props.point.image + ') center / cover'
     }
+    let seeButton = (
+      <Button colored onClick={this.onSeeMore.bind(this)}>See More</Button>
+    );
+    if (this.state.fullScreen) {
+      seeButton = (
+        <Button colored onClick={this.onSeeLess.bind(this)}>See Less</Button>
+      );
+    }
+
+    let point = this.props.point;
+    let day = this.getDays(point.hours).filter(
+      (dayEle)=>{
+        return dayMap.indexOf(dayEle.day) == (new Date()).getDay();
+      })[0]; // get the day that matches today.
+
+    // when is the service open
+    let timeDetails = (<span>{(day) ?
+        <span className="open-until"> Open until: {day.closes}</span>
+      :
+        <span className="open-until"> Not Open Today </span>
+      }</span>);
+
+    // small screen details
+    let cardDetails = (
+      <CardText>
+        {point.description}
+        {timeDetails}
+      </CardText>
+    );
+
+    let seasonal = point.hours.length > 1;
+
+    // large screen details
+    if (this.state.fullScreen) {
+      cardDetails = (
+        <div id="point-details">
+          <CardText> {point.description} {timeDetails}</CardText>
+          <CardText> {point.type} </CardText>
+          {point.amenities.map((amenity) => {
+            return(<CardText>- {amenity}</CardText>)
+          })}
+          <CardText> {point.phone} </CardText>
+          <CardText> Visit <a href={point.website}>{point.website}</a> for more details </CardText>
+          { seasonal ? <br/> : <CardText> these hours are seasonal (call or check online for more information) </CardText> }
+          <HoursTable hours={this.getDays(point.hours)}/>
+        </div>
+      )
+    }
+
     return (
-      <Card id="mdl-map-card" shadow={5} style={cardStyle}>
-        <CardTitle style={cardTitleStyle}>{this.props.point.name}</CardTitle>
-        <CardText>
-            {this.props.point.description}
-        </CardText>
-        <CardActions border>
-            <Button colored>See More</Button>
-        </CardActions>
-        <CardMenu style={{color: '#fff'}}>
-            <IconButton name="share" />
-        </CardMenu>
-      </Card>
+      <Hammer vertical={true} onPanStart={this.handleSwipe.bind(this)} onPan={this.handleSwipe.bind(this)}>
```

Comment:
should we maybe have Hammer-ified components separate from non-hammer-ified components?

this file is pretty complex, it would be nice to know we can extract the gesture logic cleanly, if we want to change the logic for doing that.